### PR TITLE
example: remove extraneous code block fence in README.md

### PR DESCRIPTION
**Pull request by stormc**
Signed-off-by: Christian Storm <christian.storm@siemens.com>

### CODE REVIEW

**Review by jan-kiszka — CHANGES_REQUESTED** [review]
All 3 or even 4 (blank) lines should go, see f2f389e8fb115e42319c7d323a9ffa2e304dd880.

File: meta-iot2000-example/README.md [hunk 1]
```diff
@@ -84,12 +84,6 @@ environment.
 This replaces the changes to conf/bblayers.conf documented in
 ()[../meta-iot2000-bsp/README.md]. The changes to conf/local.conf are the same.
```
Comment (stormc) [inline meta-iot2000-example/README.md:85]: Note: this broken link is fixed [here](https://github.com/siemens/meta-iot2000/pull/23/files#diff-83b9da2db5f8438a48c811daa6692f5fR103).

Comment (jan-kiszka) [reply meta-iot2000-example/README.md:85]: Now you probably want to adjust $subject as well.

Comment (stormc) [reply meta-iot2000-example/README.md:85]: Jap, right, done.

**Review by jan-kiszka — APPROVED** [review]

**Pull request merged**